### ISSUE TITLE
TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))

### ISSUE THREAD

**Issue by NetOperatorWibby** [silverwind/updates#34]
# updates --update ./

```error
TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))
    at checkUrlDep (/.../node_modules/updates/updates.js:541:35)
    at /.../node_modules/updates/updates.js:647:14
    at Array.map (<anonymous>)
    at main (/.../node_modules/updates/updates.js:644:66)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

# package.json

```json
{
  "dependencies": {
    "compression": "^1.7.4",
    "marked": "^0.7.0",
    "natural-orderby": "^2.0.3",
    "polka": "next",
    "sirv": "^0.4.2"
  },
  "devDependencies": {
    "@babel/core": "^7.7.4",
    "@babel/plugin-syntax-dynamic-import": "^7.7.4",
    "@babel/plugin-transform-runtime": "^7.7.4",
    "@babel/preset-env": "^7.7.4",
    "@babel/runtime": "^7.7.4",
    "@inc/eslint-config": "^2019.10.22",
    "@inc/stylelint-config": "^2019.12.1",
    "@inc/uchu": "^2019.12.1",
    "chronver": "^2019.10.2-7.1",
    "cypress": "^3.7.0",
    "eslint": "^6.7.2",
    "husky": "^3.1.0",
    "link-module-alias": "^1.2.0",
    "npm-run-all": "^4.1.5",
    "rollup": "^1.27.6",
    "rollup-plugin-babel": "^4.3.3",
    "rollup-plugin-commonjs": "^10.1.0",
    "rollup-plugin-node-resolve": "^5.2.0",
    "rollup-plugin-replace": "^2.2.0",
    "rollup-plugin-svelte": "^5.1.1",
    "rollup-plugin-terser": "^5.1.2",
    "sapper": "^0.27.9",
    "sass": "^1.23.7",
    "snazzy": "^8.0.0",
    "standardx": "^5.0.0",
    "stylelint": "^12.0.0",
    "stylelint-order": "^3.1.1",
    "svelte": "^3.16.0",
    "svelte-preprocess": "^3.2.6",
    "updates": "^9.1.0"
  }
}
```

I am not sure but I'm thinking that _maybe_ some of my modules may be conflicting with how `updates` resolves versions. Four of the modules in my `devDependencies` are my own and are using [ChronVer](https://www.npmjs.com/package/chronver) in favor of SemVer: `@inc/eslint-config`, `@inc/stylelint-config`, `@inc/uchu`, `chronver`. I also run my own registry (all but `@inc/uchu` exist on npm as well).

I'll keep investigating and report back if I find a workaround.

**Comment by NetOperatorWibby** [comment]
I'm on Node v13.2.0 btw.

**Comment by NetOperatorWibby** [comment]
I went back to version `8.5.3` but when I run my update command again (after `npm i`), I get that error again.

**Comment by silverwind** [comment]
Fixed in 9.3.1.
